test/dotnet-migrate.Tests/GivenThatIWantToMigrateSolutions.cs

```diff
@@ -204,11 +256,10 @@ private void MigrateAndBuild(string groupName, string projectName, [CallerMember
                 .Execute($"restore \"{solutionRelPath}\"")
                 .Should().Pass();
 
-            //ISSUE: https://github.com/dotnet/cli/issues/5205
```

Comment:
This appears to still fail (sporadically) on other platforms (looking at your test failures).